lms/envs/common.py

```diff
@@ -1529,8 +1529,16 @@
 
 
 common_js = set(rooted_glob(COMMON_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)    # pylint: disable=line-too-long
-project_js = set(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)  # pylint: disable=line-too-long
-
+project_js = (
+    set(
+        rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js') +
+        [
+            'common/js/xblock/runtime.v1.js',
+            'lms/js/xblock/lms.runtime.v1.js',
+        ]
+    )
+    - set(courseware_js + discussion_js + notes_js + instructor_dash_js)
```

Comment:
I've pushed up a commit that breaks this down into more clearly named variables. Let me know if you think this is enough.